bindings/nodejs/lib/types/models/info/node-info-protocol.ts

```diff
@@ -169,42 +158,80 @@ export interface WorkScoreStructure {
     minStrongParentsThreshold: number;
 }
 
+/**
+ * Mana Structure defines the parameters used by mana calculation.
+ */
+export interface ManaStructure {
+    /**
+     * The number of bits used to represent Mana.
+     */
+    bitsCount: number;
+    /**
+     * The amount of potential Mana generated by 1 IOTA in 1 slot.
+     */
+    generationRate: number;
+    /**
+     * The scaling of ManaGenerationRate expressed as an exponent of 2.
+     */
+    generationRateExponent: number;
+    /**
+     * A lookup table of epoch index diff to mana decay factor (slice index 0 = 1 epoch).
+     */
+    decayFactors: number[];
```

Comment:
Maybe using `Uint16Array` could be worthwhile? but I'm not sure if that can cause other problems elsewhere.

Reply:
yea that sounds scary :D